great_expectations/core/expectation_diagnostics/expectation_diagnostics.py

```diff
@@ -36,8 +36,6 @@
 except ImportError:
     isort = None
 
-# from pydantic.dataclasses import dataclass
-
```

Comment:
I guess we were playing around with the idea of Pydantic but deleting these since we're sticking with native dataclasses!

Reply:
Awesome! I never liked seeing those commented out pydantic lines everywhere